test/integration/singlecluster/controller/jobs/job/job_controller_test.go

```diff
@@ -3530,6 +3530,7 @@ var _ = ginkgo.Describe("Job with elastic jobs via workload-slices support", gin
 
 	ginkgo.It("Should mark old pending workload-slice evicted by scheduler as finished", func() {
 		features.SetFeatureGateDuringTest(ginkgo.GinkgoTB(), features.ElasticJobsViaWorkloadSlices, true)
+		features.SetFeatureGateDuringTest(ginkgo.GinkgoTB(), features.LocalQueueMetrics, true)
```

Comment:
I don't want to mix the ElasticJob case and LocalQueueMetrics case.
Could we have dedicated `It`?

This might be good place: https://github.com/kubernetes-sigs/kueue/blob/4c66b01c67e972c7250eeeb8ffeb87ccf961054a/test/integration/singlecluster/controller/jobs/job/job_controller_test.go#L2733